pkg/cli/cmd/deploy/deploy.go

```diff
@@ -87,6 +87,10 @@ rad deploy myapp.bicep --environment production
 # deploy using a specific environment and resource group
 rad deploy myapp.bicep --environment production --group mygroup
 
+# deploy using an environment ID and a resource group. The application will be deployed in mygroup scope, using the specified environment.
+# use this option if the environment is in a different group.
+rad deploy myapp.bicep --environment /planes/radius/local/resourcegroups/prod/providers/Applications.Core/environments/prod --group mygroup
```

Comment:
Will we be able to pass in just a name? It says here like that: https://github.com/radius-project/radius/pull/7895/files#diff-bf4e917446c73565ee6967f28bdd8e619c0d4fa40b215f195b840c021dd36ce9R104.

Reply:
Yes, we can still pass a name. In that case, the current group is searched for the environment (like before). But if we want to use an environment in a different group, then we would have to specify the ID so that we know the group.

Reply:
Why do we need the resource group?

Reply:
Can we also just do `rad deploy myapp.bicep --environment /planes/radius/local/resourcegroups/prod/providers/Applications.Core/environments/prod`?

Reply:
Also, in the example the resource group in the environment ID is different than the resource group that is passed via a flag.

We should be more specific in these examples.

Reply:
```rad deploy myapp.bicep --environment /planes/radius/local/resourcegroups/prod/providers/Applications.Core/environments/prod --group mygroup```

This command deploys the application to group `mygroup` using environment  `/planes/radius/local/resourcegroups/prod/providers/Applications.Core/environments/prod`. 
--group flag existed already and specifies the group to which the application should be deployed to. There wasnt any ability to use an environment from another group though. But if we think about it, environments contain information used for deploying apps, and user should be able to use environments from any group.